react-native-reanimated.d.ts

```diff
@@ -543,8 +543,13 @@ declare module 'react-native-reanimated' {
     isActive: boolean;
     callbackId: number;
   };
+  export type FrameTimings = {
+    timestamp: number;
+    timeSinceLastFrame: number | null;
```

Comment:
Technically, it's "duration".

![obraz](https://user-images.githubusercontent.com/20516055/188122345-b2859857-19a3-4da2-b6f2-902f5bfaee69.png)

Reply:
Thanks! 💡